services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceTest.kt

```diff
@@ -484,14 +484,14 @@ class HakemusServiceTest {
     @ExtendWith(OutputCaptureExtension::class)
     inner class HandleApplicationUpdates {
         private val alluid = 42
-        private val applicationId = 13L
+        private val applicationId = 0L
```

Comment:
Instead of setting this to zero, you could add `id` as a parameter of `HakemusFactory.createEntity()`.

Reply:
Thank you for the suggestion, I did that.